libvmaf/test/test_model.c

```diff
@@ -90,7 +90,7 @@ static char *test_json_model()
     return NULL;
 }
 
-#if VMAF_BUILT_IN_MODELS
+#if defined(VMAF_BUILT_IN_MODELS) && !defined(DISCORD_PORT)
```

Comment:
The upstream meson file is kind of messed up.

It's conditionally calling set10 for vmaf_builtin_models. set10 defines to 1 or 0 and is meant to be used instead of conditional defines. 

If we really want to be correct here and be robust to them fixing the issue we should write this as
`#if defined(VMAF_BUILT_IN_MODELS) && VMAF_BUILT_IN_MODELS && !defined(DISCORD_PORT)`

Reply:
https://mesonbuild.com/Configuration.html